web/src/utils/sort.ts

```diff
@@ -0,0 +1,9 @@
+import moment from "moment";
+
+type SortFunction<T> = (a: T, b: T) => number;
+
+export const dateSort: SortFunction<Date> = (a, b) => moment(a).diff(moment(b));
+
+export const numberSort: SortFunction<number> = (a, b) => a - b;
+
+export const stringSort: SortFunction<string> = (a, b) => a.localeCompare(b);
```

Comment:
Are these being used? Can't see where they are being used. Also I think the name "sort" might be misleading. They seem to be comparing two values, no?

Reply:
I am sorry, these are used in organisms not in molecules. I will remove these from this PR and then add them to the "organisms and pages" PR.
About sort I think that this is the way in which sorts are handled in JavaScript
Please check out this [Link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
)
Please let me know if you have a better name, just so I can use it when I add these utils to the next PR

Reply:
I removed the file from the PR

Reply:
But in the link you sent, sort will REORDER an array. Don't these SortFunctions you have made COMPARE two values? Anyways we can revisit this in your organisms & pages PR! :P